angularjs-portal-frame/src/main/webapp/portal/main/partials/welcome-modal-template.html

```diff
@@ -0,0 +1,12 @@
+<div class="welcome-popup center">
+  <h1 ng-bind-html="welcome.title"></h1>
+  <img src="{{ welcome.img }}" class="hidden-xs">
+	<div class="sr-only" tabindex="0" id="welcomeContent" aria-labelledby="welcomeContent">
+		<p>{{ welcome.accessibilityText }}</p>
+	</div>
+  <!-- <p class="visible-xs-* learn-more">Learn more at the <a href="#">KnowledgeBase.</a></p> -->
```

Comment:
That's fair.  This might be a refactor to pick up when going after the multi-page version of this, since it's going to become more plausible that some pages have learn more links and others don't.